src/Service.Tests/SqlTests/RestApiTests/Put/MsSqlPutApiTests.cs

```diff
@@ -198,6 +198,24 @@ public class MsSqlPutApiTests : PutApiTestBase
                 $"SELECT [id], [revenue], [category], [accessible_role] FROM { _tableWithSecurityPolicy } " +
                 $"WHERE [id] = 1 AND [revenue] = 2000 AND [category] = 'anime' AND [accessible_role] = 'Anonymous' " +
                 $"FOR JSON PATH, INCLUDE_NULL_VALUES, WITHOUT_ARRAY_WRAPPER"
+            },
+            {
+                "PutOneUpdateInTableWithAutoGenPKAndTrigger",
```

Comment:
`PutOneInsertInTableWithAutoGenPKAndTrigger`, this case?

Reply:
You cannot Insert via PUT in a table with autogen PK.